Framework/API/src/DetectorInfo.cpp

```diff
@@ -93,6 +93,20 @@ Kernel::Quat DetectorInfo::rotation(const size_t index) const {
   return getDetector(index).getRotation();
 }
 
+double DetectorInfo::eFixed(const size_t index) const {
+  const auto &detector = getDetector(index);
+  auto eFixedVector =
+      detector.getNumberParameter(Geometry::ParameterMap::eFixed());
+
+  double eFixed = 0.0;
+
+  if (!eFixedVector.empty()) {
+    eFixed = eFixedVector.front();
+  }
+
```

Comment:
How do we deal with changing behavior? Currently if there are multiple detectors where each has eFixed we are not using the average, I think. Instead it mostly defaults to 0.0?